aiidalab/git_util.py

```diff
@@ -120,22 +134,22 @@ def git_clone(url, commit, path):
 
 
 @dataclass
-class GitPath(os.PathLike):
+class GitPath(os.PathLike):  # type: ignore
```

Comment:
We may want to avoid to use ignore for subclassing by setting https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-disallow-subclassing-any